src/lib/CoreInstance.cc

```diff
@@ -149,6 +149,9 @@ void CoreInstance::createL1InstructionMemory(
 
 void CoreInstance::setL1InstructionMemory(
     std::shared_ptr<simeng::MemoryInterface> memRef) {
+  assert(setInstructionMemory_ &&
```

Comment:
Same as the comment for setL1DataMemory.

Reply:
Comment left on the referenced comment. It looks like user error so resolving this duplicate.